### ISSUE TITLE
loss nan

### ISSUE THREAD

**Issue by Yuyn1234** [open-mmlab/mmdetection#4036]
For your implement of RetinaNet, I use my own backbone which achieves 79.2 top-1 accuracy on ImageNet to replace ResNet. I try three times but two of them are loss-nan. Is this a normal phenomenon？ I am sure that I load the pre-trained weights.

**Comment by leopardyao** [comment]
Have you tried sth like below?
https://github.com/open-mmlab/mmdetection/issues/2739#issuecomment-631196300
https://github.com/open-mmlab/mmdetection/issues/3013#issuecomment-643739643

**Comment by GUOShuxuan** [comment]
I meet the same problem when training the RetinaNet with 8GPUs under the exact training setting in the project on COCO2017. I find that the learning rate is different from the log file. I decreased the learning rate, but it still got Nan. Did you solve the problem?

**Comment by GUOShuxuan** [comment]
> Have you tried sth like below?
> [#2739 (comment)](https://github.com/open-mmlab/mmdetection/issues/2739#issuecomment-631196300)
> [#3013 (comment)](https://github.com/open-mmlab/mmdetection/issues/3013#issuecomment-643739643)

If we change the training hyper-parameters of the models, will this affect the final results?

**Comment by Yuyn1234** [comment]
> I meet the same problem when training the RetinaNet with 8GPUs under the exact training setting in the project on COCO2017. I find that the learning rate is different from the log file. I decreased the learning rate, but it still got Nan. Did you solve the problem?

I solve this problem by using another pre-trained model. I think it probably attributes to the pre-trained. You can check whether you load the right pre-trained weights. If you make sure you load the right weights, you can retry a few more times.

**Comment by GUOShuxuan** [comment]
> > I meet the same problem when training the RetinaNet with 8GPUs under the exact training setting in the project on COCO2017. I find that the learning rate is different from the log file. I decreased the learning rate, but it still got Nan. Did you solve the problem?
> 
> I solve this problem by using another pre-trained model. I think it probably attributes to the pre-trained. You can check whether you load the right pre-trained weights. If you make sure you load the right weights, you can retry a few more times.

Thank you so much for sharing your solution. How did you change the pre-trained model? I load it from pretrained='torchvision://resnet50'. Is something wrong with this pre-trained model?

**Comment by Yuyn1234** [comment]
> > > I meet the same problem when training the RetinaNet with 8GPUs under the exact training setting in the project on COCO2017. I find that the learning rate is different from the log file. I decreased the learning rate, but it still got Nan. Did you solve the problem?
> > 
> > 
> > I solve this problem by using another pre-trained model. I think it probably attributes to the pre-trained. You can check whether you load the right pre-trained weights. If you make sure you load the right weights, you can retry a few more times.
> 
> Thank you so much for sharing your solution. How did you change the pre-trained model? I load it from pretrained='torchvision://resnet50'. Is something wrong with this pre-trained model?

I use my own backbone. Models from torchvision should be ok. I have trained the RetinaNet-ResNet-50 successfully with 8 2080Ti GPUs using default settings.

**Comment by sarmientoj24** [comment]
@ydhong1998 i am also getting loss: inf using RetinaNet and not with FRCNN. I am using 0.01 lr.
